### ISSUE TITLE
Disable everything except tab completion

### ISSUE THREAD

**Issue by 3N4N** [dahlbyk/posh-git#927]
Because of posh-git pwsh takes 818ms to start whereas normally it takes about 500ms. I only use it for tab completion and am thinking if the other whistles are disabled then it should load faster.

How do I disable _everything_ except tab-completion?

**Comment by DominikJaniec** [comment]
If anyone is interested in such feature, I've explored that possibility at my fork, which I would like to bring here too in some near feature with #948:
* https://github.com/DominikJaniec/posh-git/tree/feature/no-promp

I didn't test it much, but it's starting faster, and as far as I've tried, it works 🐱‍💻

**Comment by 3N4N** [comment]
Wish the devs and the users were more active.  Is there any other git-related module powershell users use?  Cause the startup lag for posh-git must be unbearable for the users to use it without issue.

**Comment by DominikJaniec** [comment]
That's true, the whole `PowerShell` startup is very slow 😕 However, I'm kind of passing by that issue, with something like _"startup-log"_, which tricks my brain to not feel the pain:
![image](https://user-images.githubusercontent.com/3832797/222972264-401f01e7-e6d9-45e9-9386-7995598bdf2b.png)

Also, I'm trying to not use _"my-fancy-profile"_ when I'm not needed it, like within VS Code, or while running some scripts 😉
